morgoth/addons/api/views.py

```diff
@@ -31,6 +31,21 @@ class AddonGroupViewSet(ModelViewSet):
         permissions.DjangoModelPermissionsOrAnonReadOnly,
     ]
 
+    @list_route(methods=['POST'])
```

Comment:
You're using `list_route` because you want to fetch the group by the browser version, right? Is there a case where you would have the browser version but not the ID of the group you want to edit?

Maybe instead this entire viewset should be using the browser version as the pk it uses to find the addon groups?